astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -174,11 +174,11 @@ def make_transform_graph_docs(transform_graph):
     defining new transformations between these systems, but the
     pre-defined transformations should be sufficient for typical usage.
 
-    The color of an edge in the graph (i.e. the transformations between two
+    The color of an edge in the graph (i.e., the transformations between two
     frames) is set by the type of transformation; the legend box defines the
-    mapping from transform class name to color.
+    mapping from transform class name to color (in light mode).
```

Comment:
I don't like that the colors in the graph and in the legend don't match in dark mode.

Reply:
I can also just not touch this one. So that means, normal inheritance diagrams everywhere else will look dark, but this one will remain light mode no matter what. Is that fine by you?

Reply:
If dark mode can change colors on the plot then surely it can change the colors in the legend too.

Reply:
The colors are hardcoded for light mode. Dark mode uses the "invert" magic. If you know how to grab the color code for inverted value, please suggest changes. Thanks!

Reply:
@eerovaher , for the sake of fixing the rest of the diagrams but this one, I am just going to revert the change here and chuck this for future follow-up.

Reply:
I'm guessing it should be possible to invert the colors on the legend similarly to how it is done in the graphs, but I don't know CSS well enough to actually make the change immediately.

Reply:
OK lemme try one thing but if that doesn't work, I'll exclude this file from dark mode.

Reply:
I tried injection `class=` at `ul` level and also in a new `div` container I wrap around `ul` but neither seems to have any effect. In fact, they disappear in the final rendered HTML when I inspect page source of the RTD preview. Not sure what is going on here. At least I tried.

Reply:
You know what... I forgot to refresh the browser. OMG LOL. Anyways, I can try as follow-up PR. Been a long day.